### ISSUE TITLE
Unable to select all input when it's focused by pressing tab

### ISSUE THREAD

**Issue by mikemey01** [dm4t2/vue-currency-input#31]
I'm using the v-currency directive with vue-material input components:

```html
<md-field>
 <label>Home Price</label>
 <md-input v-model="price" v-currency @focus.native="$event.target.select()"></md-input>
</md-field>
```
with the v-currency directive added to the component, @focus.native does not work to select all text. If I remove the v-currency directive, the input selects all text as expected.

@click.native="$event.target.select()" _does_ work correctly when clicking into the input to bring focus, all text is selected. It's only when pressing the tab key to bring focus.

**Comment by dm4t2** [comment]
Hi, does the problem occur when you use`v-currency={distractionFree: false}`?

**Comment by mikemey01** [comment]
That works perfectly. I tried setting it at both the component level with v-currency={distractionFree: false} and using global options, both work.

However if you expand the distractionFree object as
```
globalOptions: {
    currency: 'USD',
    locale: 'en',
    distractionFree: {
      hideNegligibleDecimalDigits: false,
      hideCurrencySymbol: false,
      hideGroupingSymbol: false
    }
  }
```
@native.focus doesn't seem to work correctly to select all.

**Comment by dm4t2** [comment]
The distraction free formatting clashes with the text selection when focusing the input. 

Also 
```
distractionFree: {
  hideNegligibleDecimalDigits: false,
  hideCurrencySymbol: false,
  hideGroupingSymbol: false
}
```
should have have the same behavior as `distractionFree: false`.

I will fix that shortly in release 1.8.4.
Thanks for the ☕️ 😋
